etc/examples/973-processes-performance-bug

```diff
@@ -0,0 +1,59 @@
+build_oci_img() {
+  # RUN apk add --no-cache bash git
+  docker build "$@" - <<'EOF'
+  FROM ubuntu
+  RUN apt update && apt install -y bc curl gcc git
```

Comment:
@infokiller totally tangential but I changed this to ubuntu to make it work for me -- do you know why your archlinux version didn't work for me (on MacOS)? Have I got a stale archlinux image cached?

```
 => CACHED [2/7] RUN pacman -Sy > /dev/null                                                                                                                                                                              0.0s
 => CACHED [3/7] RUN pacman -S --noconfirm --needed coreutils curl gcc bc git > /dev/null                                                                                                                                0.0s
 => ERROR [4/7] RUN curl -fsSL 'https://sh.rustup.rs' | sh -s -- -y --profile minimal                                                                                                                                    1.3s
------
 > [4/7] RUN curl -fsSL 'https://sh.rustup.rs' | sh -s -- -y --profile minimal:
#6 0.354 info: downloading installer
#6 1.244 /tmp/tmp.VVpvUf3oyz/rustup-init: /usr/lib/libc.so.6: version `GLIBC_2.34' not found (required by /usr/lib/libgcc_s.so.1)
```

Reply:
I have no idea but a stale image seems plausible, does anything change when running `docker pull archlinux` before the script?
Anyway, obuntu is fine and probably a better choice because of its popularity. I mostly use alpine and debian/ubuntu in docker, I think I was just being lazy using archlinux...

Reply:
OK thanks, I haven't managed to fix it yet by deleting cached image layers or by `docker pull archlinux`, but will post here if I do.